packages/cli/commands/sandbox/create.js

```diff
@@ -129,6 +129,14 @@ exports.handler = async options => {
     })
   );
   let result;
+  const twirlTimer = (function() {
```

Comment:
Super basic, can probably replace this with something from cli-spinners if we want

Reply:
We should definitely make use of the existing spinnies dependency here.